interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -145,230 +145,227 @@
-                                        <input type="text" class="form-control" size='5' id='height_input_metric' value="{if $vitals->get_height() != 0}{math equation="round(number * constant,1)" number=$vitals->get_height() constant=2.54 format="%.2f"|attr}{/if}" onChange="convCmtoIn('height_input');"/>
-                                        </td>
+                                {if $units_of_measurement == 4}<tr class="hide">{else}<tr>{/if}
+                                        {if $units_of_measurement == 2}<td class="unfocus graph" id="height">{else}<td class="graph" id="height">{/if}{xlt t="Height/Length"}</td>
+                                    {if $units_of_measurement == 2}<td class="unfocus">{else}<td>{/if}{xlt t="in"}</td>
+                                    {if $units_of_measurement == 2}<td class="valuesunfocus">{else}<td class='currentvalues'>{/if}
+                                            <input type="text" class="form-control" size='5' name='height' id='height_input' value="{if $vitals->get_height() != 0}{$vitals->get_height()|attr}{/if}" onChange="convIntoCm('height_input');"/>
+                                            </td>
                                 {foreach item=result from=$results}
-                                        <td class='historicalvalues'>{if $result.height != 0}{math equation="number * constant" number=$result.height constant=2.54 format="%.2f"|text}{/if}</td>
+                                    <td class='historicalvalues'>{if $result.height != 0}{$result.height|text}{/if}</td>
                                 {/foreach}</tr>
 
-                            <tr><td class="graph" id="bps">{xlt t="BP Systolic"}</td><td>{xlt t="mmHg"}</td>
-                                <td class='currentvalues'><input type="text" class="form-control" size='5' name='bps' id='bps_input' value="{$vitals->get_bps()|attr}"/></td>
-                            {foreach item=result from=$results}
-                                <td class='historicalvalues'>{$result.bps|text}</td>
-                            {/foreach}</tr>
+                                {if $units_of_measurement == 3}<tr class="hide">{else}<tr>{/if}
+                                            {if $units_of_measurement == 1}<td class="unfocus graph" id="height_metric">{else}<td class="graph" id="height_metric">{/if}{xlt t="Height/Length"}</td>
+                                    {if $units_of_measurement == 1}<td class="unfocus">{else}<td>{/if}{xlt t="cm"}</td>
+                                            {if $units_of_measurement == 1}<td class="valuesunfocus">{else}<td class='currentvalues'>{/if}
+                                            <input type="text" class="form-control" size='5' id='height_input_metric' value="{if $vitals->get_height() != 0}{math equation="round(number * constant,1)" number=$vitals->get_height() constant=2.54 format="%.2f"|attr}{/if}" onChange="convCmtoIn('height_input');"/>
```

Comment:
Hi @bradymiller  and @tywrenn 

Can I remove the `valuesunfocus ` class, because this style will cause the right side to have the border, looks like some odd? 
![image](https://user-images.githubusercontent.com/22230889/89645867-4bdf5e00-d8ed-11ea-9712-bb0a3596e575.png)

Reply:
Yeah as long as it doesn't change anything else about it.